app/controllers/report_controller/reports/editor.rb

```diff
@@ -1552,6 +1546,18 @@ def set_form_vars
     end
   end
 
+  def categories_hash
+    cats = Classification.categories.select { |c| c.show }
+    cats.delete_if { |c| c.entries.length == 0 }  # Omit categories for which entries dropdown would be empty.
+    cats.each_with_object({}) { |c, h| h[c.name] = c.description }
+  end
+
+  def entries_hash(category_name)
+    cat = Classification.find_by_name(category_name)
```

Comment:
That's not a dynamic finder, it's [a class method](https://github.com/ManageIQ/manageiq/blob/5aa5c4adddc39d2762ea450b6d0ea6da9a8010fe/app/models/classification.rb#L330) that does a bit more, in particular it defaults to parent_id: 0 constraint which we need for a category.
Yeah the name is a bit confusing.  Do you think a PR renaming it to something like `Classification.find_category` would be a good idea?